docs/starter_readme.template.md

```diff
@@ -0,0 +1,56 @@
+<!--
+- Default example commands to `npm` because it's the stock solution to Node.js.
+-
+-
+- Include a degit code command snippet for low-depth cloning: `npx degit <url>`.
+-->
+
+# App Title
+
+<!--
+🚀 Starters are named after the stack they power up.
+🧙‍♂️ Samples are named after the features they showcase.
+-->
+
+## Features ⚡️
+
+<!--🧙‍♂️ If Sample: which features are being showcased. -->
+
+## Setup & Usage 🐣
+
+<!--
+🚀 Starters: how to use it as a template.
+🧙‍♂️ Samples: how to run it locally.
+-->
+
+### Clone the Example 🐑
+
+<!--
+🚀 Starters: if there is a CLI to use as template, offer a command.
+🚀🧙‍♂️ Starter & Samples: provide a degit comand for shallow-cloning.
+-->
+
+```sh
+npx degit xataio/examples/apps/<name-of-app> my-xata-app
+```
+
+### Link Your Xata Workspace and Run Codegen
+
+<!--
+Explain how to push schema, link project, generate types.
+
+💡 Remember about `.env` and `.xatarc` files!
+-->
+
+```sh
+npm run start:xata
```

Comment:
```suggestion
npm run xata:init
```

This needs to be adjusted, but I think `xata:init` is more semantic for our starter